### PR TITLE
Catalog of things

### PR DESCRIPTION
**Pull request by zkr024**
* Project-based on Ruby
* Participants:
  - Boaz Aduda - @Aduda-Boaz 
  - Oscar Bermudez - @zkr024 
* The app stores information from different items added.
* The app loads the information once it is open.
* Tests to the items classes
* Schema file showing the relations between the classes created for the project.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+);
+
+CREATE TABLE Book
+(
+  id SERIAL PRIMARY KEY
+  publisher VARCHAR(100)
+  cover_state VARCHAR(100)
+  FOREIGN KEY (id) REFERENCES Items(id)
+);
+
```
Comment (keneogu) [inline schema.sql:41]: Create a game table

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @zkr024 and @Aduda-Boaz,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Nice design
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] This is one of the best ruby capstone projects. Your file structures are tremendous but you forgot to add the game.rb file. Please implement the game file.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @zkr024 and @Aduda-Boaz ,

Nice job making the required changes. Unfortunately, you missed some of them but you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Each class from the [UML class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/groupl-capstone/images/catalog_of_my_things.png) should be created in a separate .rb file, So please add `game.rb` file.

image

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by zkr024** [comment]
**@Omar-Muhamad Abut we're only 2 on the team. That is a requirement for the 3rd member. Do we still have to do it?**

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👍🏾, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**